docs/pages/versions/unversioned/config/babel.mdx

```diff
@@ -0,0 +1,162 @@
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+  };
+};
+```
+
+## Envrionment Variables
+
+Projects using `babel-preset-expo`, have access to the following environment variables:
+
+- `process.env.EXPO_OS`: The operating system of the device. This can be used as a replacement for `Platform.OS`. One of `ios`, `android`, `web`.
+- `process.env.NODE_ENV`: The environment that the project is running in. One of `development`, `production`.
+
+> You should always opt to use `process.env.NODE_ENV === 'development'` instead of `__DEV__` as this will provide support for more environments (e.g. Node.js/SSR).
+
+## babel-preset-expo
+
+This preset extends the default React Native preset (`metro-react-native-babel-preset`) and adds support for decorators, tree-shaking web packages, and loading font icons with optional native dependencies if they're installed.
+You can use this preset in any React Native project as a drop-in replacement for `metro-react-native-babel-preset`.
+
+## Options
+
+Options can be passed to the preset by passing an object as the second argument to the preset.
+
+```js babel.config.js
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: [
+      [
+        'babel-preset-expo',
+        {
+          /* options go here */
+        },
+      ],
+    ],
+  };
+};
+```
+
+### [`jsxRuntime`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#runtime)
+
+`classic | automatic`, defaults to `automatic`
+
+- `automatic` automatically convert JSX to JS without the need to `import React from 'react'` in every file. Be sure to follow the rest of the [setup guide](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#how-to-upgrade-to-the-new-jsx-transform) after enabling this, otherwise ESLint and other tools will throw warnings.
+- `classic` does not automatically import anything, React must imported into every file that uses JSX syntax.
+
+```js
+[
+  'babel-preset-expo',
+  {
+    jsxRuntime: 'classic',
+  },
+];
+```
+
+This property is passed down to [`@babel/plugin-transform-react-jsx`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx). This flag does nothing when `native.useTransformReactJSXExperimental` is set to `true` because `@babel/plugin-transform-react-jsx` is omitted.
+
+### [`jsxImportSource`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#importsource)
+
+`string`, defaults to `react`
+
+This option allows specifying a custom import source for importing functions.
+
+```js
+[
+  'babel-preset-expo',
+  {
+    jsxRuntime: 'automatic',
+    jsxImportSource: 'react',
+  },
+];
+```
+
+This property is passed down to [`@babel/plugin-transform-react-jsx`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx). This options does nothing when `jsxRuntime` is not set to `automatic`.
+
+### [`lazyImports`](https://babeljs.io/docs/en/babel-plugin-transform-modules-commonjs#lazy)
```

Comment:
Not sue it this link will work or if it will, if it will be correct. In general it's recommended to not use links inside of the headings.